packages/core/src/browser/saveable.ts

```diff
@@ -24,7 +24,7 @@ import { waitForClosed } from './widgets';
 
 export interface Saveable {
     readonly dirty: boolean;
-    readonly onDirtyChanged: Event<void>;
+    readonly onDidDirtyChange: Event<void>;
```

Comment:
@danarad05 can we revert this change, it is an unnecessary breaking change that is out of scope of the pull-request.
I believe @RomanNikitenko was only suggesting updating the new events introduced by the pull-request in order to align with our coding guidelines.

Reply:
sure, I meant we should follow the pattern from the coding guidelines for the **new** events.
I'm sorry, if I misled you, Dan...

Reply:
Now I see, it's due to:

```
class MainCustomEditorModel implements CustomEditorModel
```
and 

```
interface CustomEditorModel extends Saveable
```

so, my comment for `class MainCustomEditorModel` led to unnecessary change for  `Saveable`

Dan, please revert it...

Reply:
another question here:
 why do we need duplicate `readonly onDirtyChanged: Event<void>;` in `interface CustomEditorModel`
if: 
- `interface CustomEditorModel extends Saveable`
and 
- `interface Saveable` contains `readonly onDirtyChanged: Event<void>;`